```
/home/runner/work/mod-account-mounts/mod-account-mounts/modules/mod-account-mounts/src/mod_account_mount.cpp:18:53: fatal error: 'GetStringDefault' is deprecated: Use GetOption<std::string> instead [-Wdeprecated-declarations]
        std::string excludedSpellsStr = sConfigMgr->GetStringDefault("Account.Mounts.ExcludedSpellIDs", "");
                                                    ^
/home/runner/work/mod-account-mounts/mod-account-mounts/src/common/Configuration/Config.h:57:7: note: 'GetStringDefault' has been explicitly marked deprecated here
    [[deprecated("Use GetOption<std::string> instead")]]
      ^
1 error generated.
make[2]: *** [modules/CMakeFiles/modules.dir/build.make:104: modules/CMakeFiles/modules.dir/mod-account-mounts/src/mod_account_mount.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1199: modules/CMakeFiles/modules.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/src/mod_account_mount.cpp
+++ b/src/mod_account_mount.cpp
@@ -15,7 +15,7 @@ public:
     AccountMounts() : PlayerScript("AccountMounts")
     {
         // Retrieve the string of excluded Spell IDs from the config file
-        std::string excludedSpellsStr = sConfigMgr->GetStringDefault("Account.Mounts.ExcludedSpellIDs", "");
+        std::string excludedSpellsStr = sConfigMgr->GetOption<std::string>("Account.Mounts.ExcludedSpellIDs", "");
         // Proceed only if the configuration is not "0" or empty, indicating exclusions are specified
         if (excludedSpellsStr != "0" && !excludedSpellsStr.empty())
         {

